### PR TITLE
Update DG and UG to include autocomplete feature

### LINKED ISSUE #93: Update DG and UG to include autocomplete feature

### PR DESCRIPTION
**Pull request by isaactodo**
Closes #93

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -276,25 +277,26 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* *`    | educator | record students grades for tests and assignments          |                                                                |
+| `* *`    | educator | view a summary of each student's grade                    |                                                                |
+| `* *`    | educator | tag students with relevant labels                         | prioritize students based on their status                      |
+| `* *`    | educator | record notes on student behaviour                         | easily track issues related to their behaviour                 |
+| `* *`    | educator | archive old student data                                  | keep my AB clean while being able to retrieve old information  |
+| `*`      | educator | set learning goals                                        | track their progress towards these goals                       |
+| `*`      | educator | undo/redo any changes                                     | avoid re-entering the data during a mis-entry                  |
+| `*`      | educator | see a list of suggested commands when typing out commands | easily                                                         |
```
Comment (yongqqqq) [inline docs/DeveloperGuide.md:296]: I think there is a incomplete sentence here. 

"easily"

**Review by yongqqqq — APPROVED** [review]
Overall good code!

LGTM!

**Review by RadieonAjax — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (isaactodo) [reply docs/DeveloperGuide.md]: Fixed!

**Pull request merged**